api/src/services/UsersService.ts

```diff
@@ -14,8 +14,7 @@ class UsersService {
   static async create(body: CreateUserInput): Promise<any> {
     const { password, email, firstName, lastName } = body;
     const hashedPassword = await AuthService.geHashedPassword(password);
-    const usersRepo = new UsersRepository();
```

Comment:
I think all repositories should have static methods because they don't need session context. That logic should be in service layer.

Reply:
Yes, we do not store any info in the repo class. So we can keep it static